demo-extended/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,10 @@
       </intent-filter>
     </activity>
 
+    <receiver
```

Comment:
Because it doesn't specify any intent filters, the `exported` values is false by default. [Android documentation](http://developer.android.com/guide/topics/manifest/receiver-element.html#exported)